samples/csharp/getting-started/AnomalyDetection_CreditCardFraudDetection/CreditCardFraudDetection.Predictor/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.IO;
+
+using CreditCardFraudDetection.Common;
+
+namespace CreditCardFraudDetection.Predictor
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string assetsPath = GetAbsolutePath(@"../../../assets");
+            string trainOutput = GetAbsolutePath(@"../../../../CreditCardFraudDetection.Trainer/assets/output");
+
+            CopyModelAndDatasetFromTrainingProject(trainOutput, assetsPath);
+
+            var inputDatasetForPredictions = Path.Combine(assetsPath, "input", "testData.csv");
+            var modelFilePath = Path.Combine(assetsPath, "input", "randomizedPca.zip");
+
+            // Create model predictor to perform a few predictions
+            var modelPredictor = new Predictor(modelFilePath, inputDatasetForPredictions);
+
+            modelPredictor.RunMultiplePredictions(numberOfPredictions: 5);
+
+            Console.WriteLine("=============== Press any key ===============");
+            Console.ReadKey();
+        }
+
+
+        public static void CopyModelAndDatasetFromTrainingProject(string trainOutput, string assetsPath)
+        {
+            if (!File.Exists(Path.Combine(trainOutput, "testData.csv")) ||
+                !File.Exists(Path.Combine(trainOutput, "randomizedPca.zip")))
+            {
+                Console.WriteLine("***** YOU NEED TO RUN THE TRAINING PROJECT FIRST *****");
```

Comment:
Is there a reason we don't include a copy of randomizedPca.zip for the user already (i.e. is it too large of a file to upload to GitHub)? For the other samples, I believe we include a copy of the model/files produced from training, and the user doesn't have to train first themselves to get the predictor to work.

Reply:
I based this on how it was done in the [BinaryClassification version of this sample](https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/BinaryClassification_CreditCardFraudDetection).  Although, it actually looks like that sample does [commit the model](https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/BinaryClassification_CreditCardFraudDetection/CreditCardFraudDetection.Predictor/assets/input) into the input directory of the sample's Predictor project.  However, as mentioned above, both the test data csv and the model are generated by the Trainer project (in both samples) and looks like the BinaryClassification version of this sample doesn't commit that csv either.  So even in the BinaryClassification sample, the Trainer project will need to be run before the Predictor sample will work.

I'm happy to follow you guidance here.  If we want to commit the model and test data set into the Predictor project, I can definitely make the change, but we should probably do it for both samples.  I believe the longer-term goal is to have all these samples run as a single project vs. separate Trainer/Predictor projects.  But again, that's a change that should be made in both the CreditCardFraudDetection samples, and may be outside the scope of this PR.

Let me know how you want to move forward.

Reply:
@CESARDELATORRE Would love to get your thoughts here!

Reply:
This concrete sample has two projects. The predictor/scoring project can have the model.ZIP file. But the training project doesn't need to have it and since the code is split in two projects it is even better that the training project doesn't have it.
We can merge it as it is now. :)

Reply:
Btw, never mind. For this case it might be better if the scoring/client app copies the model so it takes the latest training. We'll change the code so it is only copying the .zip model and the test dataset. The scoring project doesn't need the training dataset and git ignore files that are being copied currently. But this is a minor improvement for clarity. 

Thanks! 👍